imagetool/src/main/java/com/oracle/weblogic/imagetool/cachestore/FileCacheStore.java

```diff
@@ -123,9 +134,40 @@ private void loadProperties(File propsFile) {
                 tmpProperties.forEach((key, value) -> properties.put(((String) key).toLowerCase(), value));
             }
         } catch (IOException e) {
-            e.printStackTrace();
+            // it is okay to fail, the constructor will attempt to create a new one
+            logger.fine("Failed to load properties file", e);
```

Comment:
Not throwing exception?

Reply:
No, not throwing an exception.  If it cannot read the property file (cache metadata), it tries to build a new one.  If that fails, then it throws the exception and exits.